### PR TITLE
Healthy food scoring

### PR DESCRIPTION
**Pull request by chompasina**
- Creates hit box around tray
- Adds score to the page
- Should add tests in next iteration

### CODE REVIEW

File: lib/health-food.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ function HealthFood(settings = {}) {
   this.y      = settings.y || 0;
   this.height = settings.height || 20;
   this.width  = settings.width || 20;
-  this.color  = settings.color || 'orange';
+  this.color  = settings.color;
   this.type   = settings.type  || 'carrot';
```
Comment (karinamzalez) [inline lib/health-food.js]: should we also take out the default type?

File: lib/junk-food.js [hunk 2]
```diff
@@ -3,7 +3,7 @@ function JunkFood(settings = {}) {
   this.y      = settings.y || 100;
   this.height = settings.height || 20;
   this.width  = settings.width || 20;
-  this.color  = settings.color || 'yellow';
+  this.color  = settings.color;
   this.type   = settings.type  || 'cheeto';
```
Comment (karinamzalez) [inline lib/junk-food.js]: maybe here as well?

File: lib/index.js [hunk 3]
```diff
@@ -102,3 +104,29 @@ function junkFoodMove(junkFood){
...
  junkFood.y += junkFood.speed;
 }
+
+function isHealthFoodCollidingWithTray(healthFood, tray){
+    return !(
+      tray.x > healthFood.x + healthFood.width || 
+      tray.x + tray.width < healthFood.x ||
+      tray.y > healthFood.y + healthFood.height ||
+      tray.y + tray.height < healthFood.y); 
+      // }
```
Comment (karinamzalez) [inline lib/index.js]: make note to delete this comment

**Review by karinamzalez — APPROVED** [review]
looks pretty darn good!

**Comment by deborahleehamel** [comment]
Should we write a feature test for this since we can't really do unit testing? @neight-allen

**Pull request merged**